README.md

```diff
@@ -1,34 +1,251 @@
-Setting up a "Hello Microverse" project
-
-Table of contents
-  About the proyect
-   the hello microverse proyect is to get to know the use of git hub, linters and recibe review
-   Built  with
-    Tech Stack
-     build html abd css on Vscode 
-    Key Features
-
-    Live demo
-        click on link  https://gonzaloalonzo.github.io/hellomicroverse/
-Getting Started
-    setup
-    Prerequisites
-    Install
-    Usage
-    Run Test
-    Deployment
-
-Authors
+
+<a name="readme-top"></a>
+
+
+<div align="center">
+  
+  <img src="murple_logo.png" alt="logo" width="140"  height="auto" />
+  <br/>
+
+  <h3><b>Microverse README Template</b></h3>
+
+</div>
+
+
+
+# 📗 Table of Contents
+
+- [📖 About the Project](#about-project)
+    the hello microverse proyect is to get to know the use of git hub, linters and recibe review
+  - [🛠 Built With](#built-with)
+        I use VScode
+    - [Tech Stack](#tech-stack)
+        Html an css
+    - [Key Features](#key-features)
+        first feature 
+  - [🚀 Live Demo](#live-demo)
+     https://gonzaloalonzo.github.io/hellomicroverse/
+- [💻 Getting Started](#getting-started)
+  - [Setup](#setup)
+  Follow the link that you can open in any browser
+  - [Prerequisites](#prerequisites)
+    You only need a browser
+  - [Install](#install)
+    No need to install
+  - [Usage](#usage)
+    it use for testing linders and github
+  - [Run tests](#run-tests)
+    se realizan test the linders
+  - [Deployment](#triangular_flag_on_post-deployment)
+    none
+- [👥 Authors](#authors)
     Gonzalo Alonzo
-Future Features
+- [🔭 Future Features](#future-features)
+    no future Features
+- [🤝 Contributing](#contributing)
+    Microverse
+    @bellom
+- [⭐️ Show your support](#support)
+    Thanks to @bellom
+- [🙏 Acknowledgements](#acknowledgements)
+- [❓ FAQ (OPTIONAL)](#faq)
+- [📝 License](#license)
+
+<!-- PROJECT DESCRIPTION -->
+
+# 📖 [hello microverse] <a name="hellomicroverse"></a>
+
+> use of html and css a simple web page
+
+**[helo micriverse]** is a test page
+
+## 🛠 Built With <a name="hellomicroverse"></a>
+
+### Tech Stack <a name="tech-stack"></a>
+
+> Describe the tech stack and include only the relevant sections that apply to your project.
+
+<details>
+  <summary>Client</summary>
+  <ul>
+    <li><a href="https://reactjs.org/">React.js</a></li>
+  </ul>
+</details>
+
+<details>
+  <summary>Server</summary>
+  <ul>
+    <li><a href="https://expressjs.com/">Express.js</a></li>
+  </ul>
+</details>
+
+<details>
+<summary>Database</summary>
+  <ul>
+    <li><a href="https://www.postgresql.org/">PostgreSQL</a></li>
+  </ul>
+</details>
+
+<!-- Features -->
+
+### Key Features <a name="key-features"></a>
+
+> Describe between 1-3 key features of the application.
+
+- **[Use of html and css]**
+- **[linters]**
+- **[github]**
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- LIVE DEMO -->
+
+## 🚀 Live Demo <a name="live-demo"></a>
+
+> Add a link to your deployed project.
+
```

Comment:
- [ ] The last reviewer asked that you remove statements like this in the README.md file, kindly remove them because they are placeholder texts that give you an idea of what you should do in each section of the README.md file. 👍 😎